tonic/src/codec/encode.rs

```diff
@@ -66,13 +20,9 @@ where
 ///  * The encoded buffer surpasses YIELD_THRESHOLD.
 #[pin_project(project = EncodedBytesProj)]
 #[derive(Debug)]
-pub(crate) struct EncodedBytes<T, U>
-where
-    T: Encoder<Error = Status>,
-    U: Stream<Item = Result<T::Item, Status>>,
-{
+pub(crate) struct EncodedBytes<T, U> {
```

Comment:
This visibility can also be removed as this type is not used outside of this module.

Reply:
Good call, removed.